src/shared/sandboxRequest.ts

```diff
@@ -88,7 +88,10 @@ export async function createSandboxRequest(
 
   const capitalizedVarArgs = properties ? lowerToUpper(properties) : {};
   // varargs override file input
-  const sandboxReqWithName: SandboxRequest & { SourceSandboxName?: string; SourceId?: string } = {
+  const sandboxReqWithName: (SandboxRequest & { Features?: string }) & {
+    SourceSandboxName?: string;
+    SourceId?: string;
+  } = {
```

Comment:
`SourceId` is already a prop of SandboxRequest.  Is `SourceSandboxName` the same as `SandboxName`?  We should try to keep the types defined in 1 place and avoid intersections unless we have to.